src/root/config.js

```diff
@@ -1,6 +1,7 @@
 export const environment = process.env.REACT_APP_ENVIRONMENT || process.env.NODE_ENV || 'development';
 export const api = process.env.REACT_APP_API_ENDPOINT ||
```

Comment:
We probably needed this check because this is being used for local development I believe, but I must be wrong. @batpad Sorry don't have the time to test right now but please make sure we're not messing with prod data locally even by accident.

Reply:
@kamicut could you look? happy to discuss.

Reply:
@GregoryHorvath this looks okay to me, can we deploy this to staging and test? This doesn't seem to change the fundamental logic of things apart from changing the URL.

Reply:
@batpad Sure we can merge, but I'd say still keep an eye on local development. If that's untouched we're good, but I highly suspect because of this change also the local development will point to the staging DB, but again I didn't have the time to test this.